src/pocket/PocketSynthetic.ts

```diff
@@ -0,0 +1,224 @@
+import { Resource } from 'cdktf';
+import { Synthetics, IAM, CloudWatch } from '@cdktf/provider-aws';
+import { Construct } from 'constructs';
+import { ArchiveProvider, DataArchiveFile } from '@cdktf/provider-archive';
+
+// the exported type is called compile? I assume
```

Comment:
I just did a few google searches and couldn't find a built-in named `compile`. What happens when you name the variable `compile`?